buildSrc/src/main/groovy/org/apache/beam/gradle/DependenciesPlugin.groovy

```diff
@@ -0,0 +1,265 @@
+/*
```

Comment:
@kennknowles I think you've been wanting to split up BeamModulePlugin.groovy for a while, does this seem like a reasonable separation to you?

Reply:
Spoke about this with Kenn offline, he suggested that this doesn't need to be a plugin, it could just be a class, similar to  `GrpcVendoring_1_26_0`.

Reply:
Moved to a simple class, looks much better, thanks

Reply:
It looks like this broke the build. I think you'll still need to have some logic in BeamModulePlugin.groovy to set the `project.ext.library`